src/operators/rasterscaling.md

```diff
@@ -69,7 +70,7 @@ The `MetadataKeyOrConstant` type is used to specify a metadata key or a constant
       "value": 1.0
     },
     "outputMeasurement": null,
-    "scalingMode": "scale"
+    "scalingMode": "mulSlopeAddOffset"
```

Comment:
in line 69 the type should be "constant", not "value"

Reply:
[7e99849](https://github.com/geo-engine/docs/pull/42/commits/7e9984997a37776e267dafeee094533e50793f17)